website/docusaurus.config.js

```diff
@@ -172,6 +172,20 @@ module.exports = {
                 id: 'GTM-5P7MCS7',
             },
         ],
+        [
+            '@signalwire/docusaurus-plugin-llms-txt',
+            {
+                enableDescriptions: false,
+                content: {
+                    excludeRoutes: ['/js/api/3.*/**', '/js/api/3.*', '/js/api/next/**'],
```

Comment:
maybe you'll also need `'/js/api/next'`, i think it wont be matched by `'/js/api/next/**'`